doc/multiproperty.md

```diff
@@ -0,0 +1,60 @@
+# Multi-property verification
+
+Multi-property verification is aimed at checking several properties
+in one verification run and providing status for each property.
+For more information about multi-property verification
+please read our paper [On-The-Fly Decomposition of Specifications in Software Model Checking](https://www.sosy-lab.org/~dbeyer/spec-decomposition/2016-FSE.On-the-Fly_Decomposition_of_Specifications_in_Software_Model_Checking.pdf).
+
+### Set multi-property verification mode
+In order to use multi-property verification in `benchexec` 
+tag `<multiproperty>` should be set to `true`.
+Note that verification tool also should support multi-property verification
+in order to provide any meaningful results.
+If this flag is not set (by default) then all specified properties 
+will be considered as a single composite property.
+
+### Specify several properties
+The property file (which is specified by `<propertyfile>` tag) 
+may contain several properties, for example:
+
+    CHECK( init(main()), LTL(G valid-free) )
+    CHECK( init(main()), LTL(G valid-deref) )
+    CHECK( init(main()), LTL(G valid-memtrack) )
+
+In this case property name is equal to corresponding SV-Comp category:
+`valid-deref`, `valid-free`, `valid-memtrack`, `no-overflow`, `no-deadlock`, `termination`.
+
+Also different error functions may be used in `unreach-call` properties:
+
+    CHECK( init(main()), LTL(G ! call(error_function_1())) )
+    CHECK( init(main()), LTL(G ! call(error_function_2())) )
+
+In this case property name contains error function name: `unreach-call,error_function_1`.
+
+In general case it is possible to specify any combination of properties in the property file.
+Note that verification tool may not support each combination.
+
+### Provide results in xml file
+For each given property `benchexec` provides separated status as a result
+in the following format:
+
+    <column title="status (<property_name>)" value="true|false|unknown"/>
+
+In case of global errors (parsing failed, out of memory, etc.) each property
+gets unknown status.
+
+### Get correct results
+Correct results for each task are placed in YAML configuration file
+`<task_file_name>.yml` in the following format:
```

Comment:
I don't really like this implicit coupling of input file and yaml file. So far we avoid such implicit name requirements in the input of benchexec, and make all names explicit. Would this be good here as well?

If the yaml file is what defines a task, shouldn't the yaml file be also specified as the task, and benchexec should read the input files from it?

I guess this is what #213 is about and where @delcypher might want to comment.